riotfile.py

```diff
@@ -316,8 +316,8 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                     pys=MAX_PYTHON_VERSION,
                 ),
                 Venv(
-                    name="tracer-legacy-atrrs",
-                    pkgs={"cattrs": "<23.2.0", "attrs": "==20.1.0"},
+                    name="tracer-legacy-attrs",
+                    pkgs={"cattrs": "<23.2.0", "attrs": "==22.1.0"},
```

Comment:
NB: How did you figure out that this was the issue from those example links?

Reply:
@erikayasuda I didn't - this is basically just a mergeable guess